docs/index.md

```diff
@@ -0,0 +1,117 @@
+| plugin-test-project | prd         | DOPPLER_CONFIG      | prd                   |
+| plugin-test-project | prd         | DOPPLER_PROJECT     | plugin-test-project   |
+| plugin-test-project | prd         | DOPPLER_ENVIRONMENT | prd                   |
++---------------------+-------------+---------------------+-----------------------+
+```
+
+## Documentation
+
+- **[Table definitions & examples →](/plugins/turbot/doppler/tables)**
+
+## Quick start
+
+### Install
+
+Download and install the latest Doppler plugin:
+
+```sh
+steampipe plugin install doppler
+```
+
+### Credentials
+
+| Item        | Description                                                                                                                                                                 |
+| ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Credentials | Doppler requires an [Access Token](https://docs.doppler.com/reference/auth-token-formats)                                                                                   |
+| Permissions | The permission scope of access tokens is limited to the projects or environments based on token type.                                                                       |
+| Radius      | Each connection represents a single doppler Installation.                                                                                                                   |
+| Resolution  | 1. Credentials explicitly set in a steampipe config file (`~/.steampipe/config/doppler.spc`)<br />2. Credentials specified in environment variables, e.g., `DOPPLER_TOKEN`. |
+
+### Configuration
+
+Installing the latest doppler plugin will create a config file (`~/.steampipe/config/doppler.spc`) with a single connection named `doppler`:
+
+Configure your account details in `~/.steampipe/config/doppler.spc`:
+
+```hcl
+connection "doppler" {
+  plugin = "doppler"
+
+  # `doppler_token` (required) - To create an access token, refer to https://docs.doppler.com/docs/service-tokens.
+  # Can also be set with the DOPPLER_TOKEN environment variable.
+  # doppler_token = "dp.pt.abcdVDI7jCoV92ylJS9yXYZO5CZRiGm0WWWnZgsZZih"
+
+  # `project_id` (required) - The doppler Project ID within a workplace.
+  # Can also be set with the DOPPLER_PROJECT_ID environment variable.
+  # project_id = "plugin-test-project"
+}
+```
+
+## Configuring Doppler Credentials
+
+### Access Token Credentials
+
+You may specify the Access Token to authenticate:
+
+- `doppler_token`: Specify the access token, either a personal or service token.
+- `project_id`: Specify the doppler Project ID in a workplace.
+
+```hcl
+connection "doppler" {
+  plugin = "doppler"
+  doppler_token = "dp.pt.abcdVDI7jCoV92ylJS9yXYZO5CZRiGm0WWWnZgsZZih"
+  project_id = "plugin-test-project"
+}
```

Comment:
Do we need this again?

Reply:
In other plugins(Godaddy, AWS etc..) we have added this block of statements.